src/components/ChartTitle.tsx

```diff
@@ -68,9 +68,14 @@ const ChartTitle = (props: TChartTitleProps) => {
                     store={categoricalDisplay}
                     isNestedList={isNestedList}
                     searchInputClassName={searchInputClassName}
-                    onSelectItem={(x: TProcessedSymbolItem) => {
-                        if (x.symbol !== currentSymbol.symbol) {
-                            onChange(x.symbol);
+                    onSelectItem={(symbol_object: TProcessedSymbolItem, category_id: string) => {
+                        if (symbol_object.symbol !== currentSymbol.symbol) {
+                            onChange(symbol_object.symbol);
+                            state.stateChange(STATE.SYMBOL_CHANGE, {
+                                tab_market_name:
+                                    category_id === 'favorite' ? 'favorites' : getSymbolMarketCategory(symbol_object),
+                                market_type_name: symbol_object.symbol,
```

Comment:
im just wondering why are we setting market_type_name to symbol name?

Reply:
@henry-deriv it's because we need to send it in this form to rudderstack via deriv-app. I can either call this property straight the way it needs to go to analytics and have less logic in deriv-app - it's what I do know, or name it in the way we want here and rename this properties later when they are received inside deriv-app.

Reply:
sorry. i was under the impression that market_type_name referrers to derived, forex, cryptocurrencies, etc. Thats why i was confused as to why symbol is passed as market_type_name

<img width="351" alt="Screenshot 2023-09-06 at 4 14 23 PM" src="https://github.com/binary-com/SmartCharts/assets/118344354/3157ff1f-2e9a-4dfa-9f0e-b3b473930ed7">

Reply:
@henry-deriv marketing team has their own names for all this :)) hm, I think I can change it here, and rename the property in deriv-app PR before we send it to rudderstack

Reply:
sounds good